src/watcher.lisp

```diff
@@ -108,7 +113,7 @@
   (let ((thread (with-hierarchies-lock *watcher-thread*)))
     (when thread
       (with-logged-problems '(stop-hierarchy-watcher-thread :destroy-thread)
```

Comment:
The label should no longer be ``:destroy-thread``. The commit introducing ``bt-semaphore`` should have changed it.